webpack/JobWizard/steps/HostsAndInputs/HostPreviewModal.js

```diff
@@ -31,7 +33,7 @@ export const HostPreviewModal = ({ isOpen, setIsOpen, searchQuery }) => {
               rel="noreferrer"
               isInline
             >
-              {host}
+              {displayFqdnForHosts ? host : host.split('.')[0]}
```

Comment:
I would try to avoid this. You can change/add a selector that will add the `display_name`  field for each host. Then you can use it in this modal.